src/main/java/com/marek/onlinebookstore/dto/order/OrderDto.java

```diff
@@ -0,0 +1,21 @@
+package com.marek.onlinebookstore.dto.order;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.PositiveOrZero;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+public record OrderDto(
+        @Positive
+        Long id,
+        @Positive
+        Long userId,
```

Comment:
It is not a good practice to use user id in DTO. It can lead to security issues.